client/src/components/EditProfile/EditForm/EditForm.tsx

```diff
@@ -0,0 +1,131 @@
+import { Formik, Form } from 'formik';
+import * as Yup from 'yup';
+import { Box, Grid, Typography } from '@material-ui/core';
+
+import Textfield from '../../FormsUI/Textfield';
+import Select from '../../FormsUI/Select';
+import DateTimePicker from '../../FormsUI/DateTimePicker';
+import Button from '../../FormsUI/Button';
+import useStyles from './useStyles';
+
+const INITIAL_FORM_STATE = {
+  firstName: '',
+  lastName: '',
+  gender: '',
+  birthDate: '',
+  email: '',
+  phone: '',
+  address: '',
+  message: '',
+};
+
+const FORM_VALIDATION = Yup.object().shape({
+  firstName: Yup.string().required('This field cannot be empty'),
+  lastName: Yup.string().required('This field cannot be empty'),
+  gender: Yup.string().required('This field cannot be empty'),
+  birthDate: Yup.date().required('This field cannot be empty'),
+  email: Yup.string().email('Invalid email.').required('This field cannot be empty'),
+  phone: Yup.number().integer().typeError('Please enter a valid phone number').required('This field cannot be empty'),
+  address: Yup.string().required('Required'),
+  message: Yup.string(),
+});
+
+const EditForm = (): JSX.Element => {
+  const classes = useStyles();
+  const options = ['Male', 'Female', 'Other'];
+
+  return (
+    <div>
+      <Formik
+        initialValues={{
+          ...INITIAL_FORM_STATE,
+        }}
+        validationSchema={FORM_VALIDATION}
+        onSubmit={(values) => {
+          console.log(values);
+        }}
+      >
+        <Form>
+          <Grid container alignItems="center" spacing={2}>
+            <Grid item xs={12}>
+              <Typography variant="h6" align="center">
+                Edit Profile
+              </Typography>
+            </Grid>
+            <Grid item xs={12} sm={3}>
+              <Typography variant="body1" className={classes.typography}>
+                FIRST NAME
```

Comment:
Please avoid using CAPITALCASE for text, some screen readers interpret this text as abbreviations, you can resort to using a CSS class to transform text instead.